ci/repokitteh/modules/newcontributor.star

```diff
@@ -0,0 +1,28 @@
+
+# not sure if this is needed, or whether `pull_request` below has info already
+def get_pr_author_association(issue_number):
+  return github.call(
+    method="GET",
+    path="repos/envoyproxy/envoy/pulls/%s" % issue_number)["json"]["author_association"]
+
+def is_newcontributor(pull_request):
+  # check for any existing open PRs from user ?
+  return pull_request["author_association"] == "FIRST_TIME_CONTRIBUTOR"
```

Comment:
not sure if this will work  - or whether the function above is required to get author assoc